grpc/server.go

```diff
@@ -3,7 +3,10 @@ package grpc
 import (
 	"context"
 	"fmt"
+	pbcodec "github.com/dfuse-io/dfuse-eosio/pb/dfuse/eosio/codec/v1"
```

Comment:
This is not a mergeable solution, `firehose` library is chain agnostic and should not depend on a specific implementation. We would need to pass in `Modules` a function that knowns how to extract the block time.